vapor.swift

```diff
@@ -386,6 +386,41 @@ extension New {
 
 commands.append(New)
 
+// MARK: Bootstrap
+
+struct Bootstrap: Command {
+    static let id = "bootstrap"
+
+    static func execute(with args: [String], in directory: String) {
+        let binary: String = {
+            if let p = args.first {
+                return p.hasSuffix("/") ? p : p + "/"
+            } else {
+                return "./"
+            }
+        }() + "vapor"
+
+        let src = Process.arguments[0]
+        let cmd = "env SDKROOT=$(xcrun -show-sdk-path -sdk macosx) swiftc \(src) -o \(binary)"
+        do {
+            try run(cmd)
+        } catch {
+            fail("Could not compile \(src), try running the folloing command in order to debug this issue:\n\(cmd)")
+        }
+    }
+
+    static var help: [String] {
+        return [
+                   "bootstrap <directory>",
```

Comment:
Yeah, looks a bit out of place - this is what I get after Re-Indent. I’ll move it over and abstain from mashing the TAB key when near those lines ;)